functions.php

```diff
@@ -2,10 +2,12 @@
 
 function returnHtmlFromArray(array $players): string {
```

Comment:
Because you have changed this, you will need to update your unit tests for this function to match the changes you added for card numbers

Reply:
Changed the test to match and got them to be successful